src/timing.h

```diff
@@ -346,11 +353,14 @@ STATIC_INLINE void _jl_timing_suspend_destroy(jl_timing_suspend_t *suspend) JL_N
 #endif
 }
 
+#define COND_XCAT(name, line) name##line
+#define COND_CAT(name, line) COND_XCAT(name, line)
```

Comment:
```suggestion
#define XCONCAT(name, line) name##line
#define CONCAT(name, line) XCONCAT(name, line)
```

TIMING_CONCAT would also be fine